files/plugins/check_cis_audit.py

```diff
@@ -17,16 +17,42 @@
 import time
 import xml.etree.ElementTree as ElementTree
 
-
 from nagios_plugin3 import (
     CriticalError,
     WarnError,
     try_check,
 )
 
 
-AUDIT_FOLDER = "/usr/share/ubuntu-scap-security-guides"
-AUDIT_RESULT_GLOB = AUDIT_FOLDER + "/cis-*-results.xml"
+def _get_major_version():
+    """Get major version from /etc/os-release."""
+    with open(os.path.join(os.sep, 'etc', 'os-release')) as fin:
+        for line in fin:
+            if "VERSION_ID" in line:
+                value = line.strip().split("=", 1)[1]
+                return int(float(value.strip('"')))
```

Comment:
non-blocking: it would be safer and cleaner if we do version parsing and comparison using `packaging.version`.